src/template.html

```diff
@@ -4,9 +4,13 @@
 	<meta charset='utf-8'>
 	<meta name='viewport' content='width=device-width,initial-scale=1.0'>
 	<meta name='theme-color' content='#333333'>
+  <meta name='description' content="🎧 The Frontend Testing Podcast excpect(Exception). You can pronounce it without brackets. Monthly episodes. Produced by Mirjam Bäuerlein, Nikolaus Rademacher, Markus Wolf, Roman Kuba, David Müllerchen and Vanessa Böhner.">
```

Comment:
Indentation looks wrong here

Reply:
I totally forgot that default Sapper comes with tabs, not 2 spaces for tabs. thanks!